samples/CustomRouteSample.Web/project.json

```diff
@@ -10,11 +10,12 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {},
+    "net451": { },
     "netcoreapp1.0": {
       "imports": [
         "dnxcore50",
-        "portable-net451+win8"
+        "portable-net451+win8",
+         
```

Comment:
Nah, no trailing comma if you want valid JSON. (Don't need the blank line either.)